### DISCUSSION TITLE
'No cars found in account' error when trying to add integration

### DISCUSSION THREAD

**Discussion by freekbe** [pypolestar/polestar_api#51] [Q&A] [answered]
I'm using the same credentials as in the Polestar app. 
I'm able to log on here https://polestarid.eu.polestar.com/PolestarLogin/login with the same credentials. 
On that page, I don't see any reference to my car, just the details about my account (this might be the normal behavior)

When adding my login data, I get the message: Unexpected error creating api
In the log files it says: No cars found in account

Not sure if this is a bug, if Polestar changed something on their side or if there is an issue with my account.

**Comment by leeyuentuen** [comment]
the account that you login is not the main owner of the car. did you have an another login?

**Reply by loebse** [reply]
The mobile apps and the polestar account on polestar.com are two separate things. 

The apps can be connected using all keys of the car, whether you own the car or not is irrelevant. 

The polestar.com account shows the car that you actually own. If you do not see your car there, the integration will not work and you will have to reach out to Polestar to add your car.

**Reply by jesseuitto** [reply]
I am the "owner" (or main user) of the car in the app details and in the car's sofware. But I have same problem, no car is shown on that PolestarID web page.

**Reply by jesseuitto** [reply]
Note! I did find this place where you can try to transfer the ownership https://www.polestar.com/change-of-ownership/initiate-claim/vin

Unfortunately it didn't work for me.

**Comment by freekbe** [comment]
Thanks, that is helpful. 

It's a leasing car, which means the car is legally owned by the leasing company and it is probably registered that way at Polestar. 
I'll reach out to Polestar, maybe they can help me out anyway.

**Reply by Stroell** [reply, marked as answer]
> Thanks, that is helpful. 
> 
> 
> 
> It's a leasing car, which means the car is legally owned by the leasing company and it is probably registered that way at Polestar. 
> 
> I'll reach out to Polestar, maybe they can help me out anyway. 

Yes, contact the support. They helped me very quickly to get my leasing car added to the portal
